src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -131,7 +131,7 @@ public function load(array $configs, ContainerBuilder $container)
                     'macvim' => 'mvim://open?url=file://%%f&line=%%l',
                     'emacs' => 'emacs://open?url=file://%%f&line=%%l',
                     'sublime' => 'subl://open?url=file://%%f&line=%%l',
-                    'phpstorm' => 'phpstorm://open?url=file://%%f&line=%%l',
+                    'phpstorm' => 'Darwin' === PHP_OS ? 'phpstorm://open?file=%%f&line=%%l' : 'phpstorm://open?url=file://%%f&line=%%l',
```

Comment:
Not sure if that's really a good idea (useful at all) since this won't work anyway when Symfony is running in a virtual machine, will it?